docs/_resources/contributors.md

```diff
@@ -0,0 +1,124 @@
+5. 💬 **Answering questions:** Being actively involved in answering questions and providing assistance to community members is an invaluable contribution to the project.
+6. 📚 **Documentation update:** Documentation serves as a crucial resource for users, developers, and contributors alike. Keeping documentation accurate, comprehensive, and up-to-date is essential for the success of the toolkit.
+
+We appreciate your interest in contributing to the FinOps Toolkit project!
+
+### 🐛 1. Working on existing bugs
+
+If you're eager to contribute but unsure where to begin, a great starting point is to review the list of existing feedback and identify issues that align with your skills and interests. 
+
+1. Look through the list of existing bugs in the [Issues tab](https://github.com/finopsfoundation/FinOpsToolkit/issues).
+2. Assign yourself to the bug you want to work on.
+
+If this is the first time you are working on the FinOps Toolkit repository, make sure to review our [Get Started Wiki.](https://github.com/microsoft/finops-toolkit/wiki#-test-and-verify)
+
+Once you are assigned to the bug, GitHub will automatically assign the tag "Community Contributor" to inform the team responsible for that tool of your contribution. 
+
+### 🛠️ 2. Suggesting and owning a new feature, fixing an existing bug, or suggesting a change for an existing tool
+
+#### Submitting a New Issue and Willing to Fix
+
+If you have a new feature suggestion, identified a bug, or have ideas for improving existing tools, we encourage you to submit an issue. Your feedback helps us ensure that the toolkit meets the needs of its users. And if you're willing to take it a step further and contribute to implementing your suggestions, we welcome your contributions with open arms! 
+
+To do that, follow these steps: 
+
+1. Go to the [Issues tab](https://github.com/finopsfoundation/FinOpsToolkit/issues) and click on "New issue".
+2. Choose the option that best describes your contribution (bug fix, change request, or feature request).
+3. Check the box "Willing to fix" at the bottom of the page.
+4. Submit your issue.
+
+Your issue will be reviewed by the FinOps Toolkit governance team, who will reach out to you to discuss further and give you the green light to start developing. 
+
+### 3. 💡 Submitting a proposal for a new tool
+
+If you have developed a FinOps-related tool and want it to be part of the FinOps Toolkit, follow these steps:
+
+1. Send an email to [ftk-leads@microsoft.com](mailto:ftk-leads@microsoft.com) with your proposal.
+2. Include as much information as possible about your tool and how it can benefit the FinOps community.
+
+The FinOps toolkit governance committee will review your proposal and reach out to you for further discussion.
+

```

Comment:
```suggestion
If you have developed a FinOps-related tool and want it to be part of the FinOps toolkit, send the details to [ftk-leads@microsoft.com](mailto:ftk-leads@microsoft.com?subject=[FTK]+New+tool+for+the+FinOps+toolkit). Please include as much information as possible about your tool and how it can benefit the FinOps community.

The FinOps toolkit governing board will review your proposal and reach out to you for further discussion.
```